runtime-manager/modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -46,6 +46,23 @@ The version of Runtime Manager agent must support the version of Mule and Runtim
 
 Mule 3.7.x and earlier and the API gateway 2.x and earlier include an earlier version of the Runtime Manager agent, which doesn't provide support for exporting data to external analytics tools. If you are running these earlier versions, you can download and update the agent to the latest version.
 
+===  Runtime Manager Agent - Mule - Java Compatibility Map
```

Comment:
```suggestion
===  Runtime Manager Agent Compatibility
```

Reply:
It's self-evident that this is Mule & Java compatibility because of the column names, so I suggest shortening this.